cmd/tools/genrpcserverinterceptors/main.go

```diff
@@ -43,10 +43,10 @@ import (
 	"go.temporal.io/server/api/matchingservice/v1"
 )
 
-const maxRequestDepth = 5
+const maxPayloadDepth = 5
 
 type (
-	requestData struct {
+	payloadData struct {
```

Comment:
Word "payload" have specific meaning in this project (and proto messages). May be `reqResp` instead of `request`?

Reply:
How about "messageData"?

Reply:
Also confusing, but I think it is better.